Makefile

```diff
@@ -764,3 +764,32 @@ include tests/integration/tests.mk
 .PHONY: benchcheck
 benchcheck:
 	bin/perfcheck.sh
+
+#-----------------------------------------------------------------------------
+# Target: istio-ecosystem installer
+#-----------------------------------------------------------------------------
+
+export ISTIO_INSTALLER?=${GOPATH}/src/github.com/istio/installer
+INSTALLER_OPTS="-f install/kubernetes/helm/istio/test-values/values-installer-e2e.yaml --set global.istioNamespace=istio-system --set global.hub=${HUB} --set global.tag=${TAG}"
+INSTALLER_REPOSRC=git://github.com/istio/installer.git
+
+force: ;
+
+${GOPATH}/src/github.com/istio/installer: force
+	mkdir -p ${GOPATH}/src/istio-ecosystem
+	git clone -b master "${INSTALLER_REPOSRC}" "${ISTIO_INSTALLER}" 2> /dev/null || (cd "${ISTIO_INSTALLER}" ; git pull)
```

Comment:
Will this just skip it if it already exists?

Reply:
if it already exists, it should pull from master.

Reply:
I am just a little worried that I'll be doing work on the installer and run this and have everything wiped out/in a bad state. not sure if there is a better solution though

Reply:
Yup, this is a decent concern... We need a place to put this that doesn't interfere with everything, ultimately.

Reply:
git submodule?